org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/EventConstants.java

```diff
@@ -13,8 +13,8 @@
 
 
 /**
- * This class represents 
- * 
+ * This class represents
+ *
  * @author Matthew Gerring
  */
 public interface EventConstants {

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)